src/main/scala/com/google/cloud/genomics/spark/examples/GenomicsConf.scala

```diff
@@ -37,23 +37,20 @@ class GenomicsConf(arguments: Seq[String]) extends ScallopConf(arguments) {
   val outputPath = opt[String]()
   val references = opt[String](default=Some(Contig.BRCA1),
       descr = "Comma separated tuples of reference:start:end,...")
-  val sparkMaster = opt[String](default = Some("local[2]"))
-  val sparkPath = opt[String](default = Some(""))
+  val sparkMaster = opt[String](
+      descr = "A spark master URL. Leave empty if using spark-submit.")
   val variantSetId = opt[String](
     default = Some(GoogleGenomicsPublicData.Thousand_Genomes_Phase_1),
       descr = "VariantSetId to use in the analysis.")
 
   def newSparkContext(className: String) = {
-    val jarPath = if (this.jarPath.isDefined)
-      Seq(this.jarPath())
-    else
-      Seq[String]()
     val conf = new SparkConf()
-      .setMaster(this.sparkMaster())
       .setAppName(className)
-      .setSparkHome(this.sparkPath())
-      .setJars(jarPath)
       .set("spark.shuffle.consolidateFiles", "true")
+    if (this.jarPath.isDefined)
```

Comment:
Is --jarPath still needed?  From the spark-submit paths above, that command line option is not explicit.